tests/integration-service/integration.go

```diff
@@ -92,6 +96,18 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 			It("checks if all of the integrationPipelineRuns passed", Label("slow"), func() {
 				Expect(f.AsKubeDeveloper.IntegrationController.WaitForAllIntegrationPipelinesToBeFinished(testNamespace, applicationName, snapshot)).To(Succeed())
 			})
+
+			It("checks if the status of integration test si reported in the Snapshot", func() {
```

Comment:
tiny typo: `checks if the status of integration test si reported in the Snapshot` -> `checks if the status of integration test is reported in the Snapshot`